MemorialHouse/MHPresentation/MHPresentation/Source/Register/View/RegisterViewController.swift

```diff
@@ -0,0 +1,176 @@
+import UIKit
+import MHFoundation
+import Combine
+
+public final class RegisterViewController: UIViewController {
+    // MARK: - Property
+    private var viewModel = RegisterViewModel()
```

Comment:
p3
private let 으로는 못 쓰나용 ??

Reply:
P2: 생성자에서 viewmodel을 받고있는데 프로퍼티가 뷰모델 객체를 들고있는게 어색하게 느껴집니다.. 

```suggestion
    private let viewModel: RegisterViewModel
```

이게 더 낫지 않을까요??

Reply:
옹 인정합니다